frappe/templates/includes/login/login.js

```diff
@@ -272,11 +272,11 @@ var request_otp = function(r){
 	$('.login-content').empty().append($('<div>').attr({'id':'twofactor_div'}).html(
 		'<form class="form-verify">\
 			<div class="page-card-head">\
-				<span class="indicator blue" data-text="Verification">Verification</span>\
+				<span class="indicator blue" data-text="Verification">{{ _("Verification") }}</span>\
```

Comment:
Please use `__` (double underscores) instead of single underscore for translating strings in `.js` files. Also, Jinja syntax won't work here. You will have to wrap this string block with backticks (**`**)

Reply:
Fix some strings, Jinga work well in this file:
before: https://drive.google.com/file/d/1hZMTgl3KQiB85-Riy-DrBFjn1ymIQo1s/view?usp=sharing
After: https://drive.google.com/file/d/1akaDQU1b9ncYYjEm9K4nU_NCfgruPyo4/view?usp=sharing

![otp_before](https://user-images.githubusercontent.com/1592496/62098409-6e08bf80-b247-11e9-978a-18d8b431f27c.PNG)

![otp_after](https://user-images.githubusercontent.com/1592496/62098407-6e08bf80-b247-11e9-98d9-7776963bb53d.PNG)